target/portfolio-1/protests.js

```diff
@@ -0,0 +1,94 @@
+google.charts.load('current', {'packages':['geochart'],
```

Comment:
This seems like a duplicate. Sara had something similar. Talk to her and find out how she removed the file,